app/models/video.py

```diff
@@ -0,0 +1,28 @@
+from flask import current_app
+from sqlalchemy.sql.schema import DefaultClause
+from app import db
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer)
+
+    rentals = db.relationship("Rental", backref="video", lazy=True)
+
+    def to_dict(self):
+        return {
+            "id": self.video_id, 
+            "title": self.title,
+            "release_date": self.release_date,
+            "total_inventory": self.total_inventory,
+            "available_inventory": self.available_inventory,
```

Comment:
Like with `Customer` the `available_inventory` is just the total inventory minus length of the `rentals` that haven't been checked in:

```suggestion
            "available_inventory": self.total_inventory - len([rental for rental in self.rentals if not rental.check_in_date])
```